ssn/ssn_separated/ssn.ttl

```diff
@@ -890,47 +773,41 @@ ssn:SensorInput rdf:type owl:Class ;
+                                   owl:onProperty sosa:hasResult ;
+                                   owl:someValuesFrom sosa:Result
                                  ] ,
                                  [ rdf:type owl:Restriction ;
                                    owl:onProperty ssn:isProducedBy ;
                                    owl:someValuesFrom sosa:Sensor
                                  ] ;
-                 dc:source """http://www.w3.org/2005/Incubator/ssn/
-
-                                  skos:closeMatch 'observation result' [O&M] 
-                                  http://www.opengeospatial.org/standards/om 
-
-See comments at ObservationValue.""" ;
-                 rdfs:comment "A sensor outputs a piece of information (an observed value), the value itself being represented by an ObservationValue." ;
+                 rdfs:comment "A Sensor outputs a piece of information (an observed value), the value itself being represented by a Result." ;
                  rdfs:isDefinedBy "http://www.w3.org/ns/ssn" ;
                  rdfs:label "Sensor Output" ;
                  rdfs:seeAlso "http://www.w3.org/2005/Incubator/ssn/wiki/SSN_Skeleton#Skeleton" .
 
+###  http://www.w3.org/ns/ssn/ActuatorOutput
 
 ###  http://www.w3.org/ns/ssn/Stimulus
 ssn:Stimulus rdf:type owl:Class ;
-             dc:source "http://www.w3.org/2005/Incubator/ssn/" ;
-             rdfs:comment "An Event in the real world that 'triggers' the sensor.  The properties associated to the stimulus may be different to the eventual observed property.  It is the event, not the object, that triggers the sensor." ;
+             rdfs:comment "An Event in the real world that 'triggers' the Sensor. The properties associated to the Stimulus may be different to the eventual ObservedProperty. It is the event, not the object, that triggers the Sensor." ;
              rdfs:isDefinedBy "http://www.w3.org/ns/ssn" ;
-             rdfs:label "Stimulus" ;
-             rdfs:seeAlso "http://www.w3.org/2005/Incubator/ssn/wiki/SSN_Skeleton#Skeleton" .
+             rdfs:label "Stimulus" .
 
 
 ###  http://www.w3.org/ns/ssn/SurvivalProperty
 ssn:SurvivalProperty rdf:type owl:Class ;
                      rdfs:subClassOf ssn:Property ;
-                     rdfs:comment "An identifiable characteristic that represents the extent of the sensor's useful life.  Might include for example total battery life or number of recharges, or, for sensors that are used only a fixed number of times, the number of observations that can be made before the sensing capability is depleted." ;
+                     rdfs:comment "An identifiable characteristic that represents the extent of the Sensor's or Actuator's useful life. Might include, for example, total battery life or number of recharges, or, for Sensors that are used only a fixed number of times, the number of observations that can be made before the sensing capability is depleted." ;
```

Comment:
An identifiable characteristic that represents the extent of the System's useful life. Might be for example BatteryLifetime  or number of recharges, or, for Sensors that are used only a fixed number of times, the number of Observations that can be made before the sensing capability is depleted.